config/Coldbox.cfc

```diff
@@ -86,34 +45,32 @@ Optional Methods
 
 		// custom settings
 		settings = {
-
 		};
 
-		// environment settings, create a detectEnvironment() method to detect it yourself.
-		// create a function with the name of the environment so it can be executed if that environment is detected
-		// the value of the environment is a list of regex patterns to match the cgi.http_host.
-		environments = {
-			//development = "^cf8.,^railo."
-		};
+		//goovy files paths
+		groovy = {libPath='/models/groovy,/models/anotherPath'};

```

Comment:
This won't work sana. It needs to be in the `settings` struct in order to be picked up by the module